src/library/scala/collection/immutable/TreeMap.scala

```diff
@@ -132,6 +132,16 @@ final class TreeMap[K, +V] private (private val tree: RB.Tree[K, V])(implicit va
     else resultOrNull.value
   }
 
+  // override for perfornance -- no Sone allocation
+  override def apply(key: K): V = {
+    val resultOrNull = RB.lookup(tree, key)
+    if (resultOrNull eq null) default(key)
+    else resultOrNull
+  }
+
+  // override for perfornance -- no Sone allocation
```

Comment:
```suggestion
  // override for perfornance -- no Some allocation
```

Reply:
and perform**m**ance.

Looks like the 'm' key was broken.